.circleci/config.yml

```diff
@@ -159,11 +159,37 @@ jobs:
     environment:
       TOX_TEST: py27
 
+  py3_compat:
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run:
+          name: install futurize
+          command: pip install future --user
+      - run:
+          name: find python3-incompatible code
+          command: |
+            FUTURIZE="futurize ."
+            while read line; do
+              FUTURIZE="${FUTURIZE} ${line}"
```

Comment:
(nit) maybe `-f` might be moved here from `py3fixers`

Reply:
@mateumann i kinda prefer this way actually, allows us to put `--stage1` in fixers later, which i'm going to do i think